modules/tinymce/src/core/test/ts/browser/SelectionOverridesTest.ts

```diff
@@ -16,9 +16,9 @@ UnitTest.asynctest('browser.tinymce.core.SelectionOverridesTest', function (succ
     LegacyUnit.equal(a, true, label);
   };
 
-  suite.test('click on link in cE=false', function (editor) {
+  suite.test('click on link in cE=false', function (editor: Editor) {
     editor.setContent('<p contentEditable="false"><a href="#"><strong>link</strong></a></p>');
-    const evt = editor.fire('click', { target: editor.$('strong')[0] } as MouseEvent);
+    const evt = editor.fire('click', { target: editor.$('strong')[0] } as any);
```

Comment:
```suggestion
    const evt = editor.fire('click', { target: editor.$<HTMLStrongElement>('strong')[0] } as MouseEvent);
```
Does that type check at all?

Reply:
No, I tried this as well. It needs `buttons` and other properties to typecheck as a `MouseEvent` and as mentioned previously I was trying to change code as little as possible.